doc.go

```diff
@@ -121,6 +121,11 @@ Application Options:
                                unless you know what you're doing
       --dumpblockchain=        Write blockchain as a flat file of blocks for use
                                with addblock, to the specified filename
+      --assumevalid=           Hash of an assumed valid block.  Defaults to the
+                               hard-coded assumed valid block that is updated
+                               periodically with new releases.  Set to 0 to
+                               disable.  Don't do this unless you know what
+                               you're doing
```

Comment:
> Don't do this unless you know what you're doing

Seems unnecessarily ominous. Isn't the only consequence of `--assumevalid=0` is that the IBD will take a lot longer? Why not: `Set to 0 to disable; however, doing so will significantly slow down the initial blockchain download.`

Reply:
I can see how that could be interpreted that way.  I took it to mean don't change the values as opposed to don't set it to 0.

It's a good point though that this setting, unlike checkpoints, doesn't have consequences as severe as the checkpoint case.

How about the following?

> Don't modify this setting unless you understand the implications.

Reply:
Ah, so the warning is specifically about changing the hash, not to 0. Rearranging then:

> Hash of an assumed valid block.  Defaults to the hard-coded assumed valid block that is updated periodically with new releases. Don't use a different hash unless you understand the implications. Set to 0 to disable.

Reply:
Agreed that there isn't as much of a risk in changing it as the wording made it out to be.  Updated to the suggested language.